src/index.ts

```diff
@@ -55,8 +56,18 @@ function init(_modules: { typescript: typeof ts_module }) {
           info.project.getCompilerOptions = function() {
             let oldOptions = oldCOptions.call(this);
             if (!patchedOptions) {
+              let oldPaths: ts_module.MapLike<string[]> =
+              Object.keys(oldOptions.paths || {}).reduce((prev, curr) => {
+                let keys = oldOptions.paths[curr].map((p: string) =>
+                  path.resolve(initialRootPath, oldOptions.baseUrl, p)
+                );
+                return Object.assign({}, prev, { [curr]: keys });
+              }, {}) || {};
```

Comment:
Doesn't need the `||` statement.
```suggestion
              }, {});
```